Misc bugfixes

BroBeansCPG opened the pull request:
**When merged this pull request will:**
Massive Exterior Sound overhaul
Misc Sound fixes 
- Fix: fuel value discrep. (fix #413)
- Fix: Sound attenuation in copilot seat
- Fix: Monocle conditional checks (monocle showing in 3rd Person & flir)
- Fix: Seats not unlocking after removing cargo (fix #385)
- Fix: FLIR not unstowing/JVMF breaking with custom cTAB versions (fix #420 & fix #379 )
- Fix: ESIS not functioning if autostarted (fix #366)
- Fix: Various build issues and config errors
- Add: Addon setting to enable/disable autostart function
- Add: ROF mode display for miniguns on DAP
- Add: Delete button for current waypoint on FMS (close #148)
- Add: New sound library

BroBeansCPG committed:
fix fms fuel count

fix #413

BroBeansCPG committed:
fix fms fuel count

fix #413

BroBeansCPG committed:
possible unstow fix

BroBeansCPG committed:
possible unstow fix

BroBeansCPG committed:
fix ROF build issue

BroBeansCPG committed:
fix ROF build issue

BroBeansCPG committed:
hide monocle where necessary

may need clean up

BroBeansCPG committed:
hide monocle where necessary

may need clean up

BroBeansCPG committed:
improve monocle check

BroBeansCPG committed:
improve monocle check

BroBeansCPG committed:
fix GVAR error

BroBeansCPG committed:
fix GVAR error

BroBeansCPG committed:
fix copilot attenuation

BroBeansCPG committed:
fix copilot attenuation

BroBeansCPG committed:
DAP minigun displayname

BroBeansCPG committed:
DAP minigun displayname

BroBeansCPG committed:
add simple startup addon option

BroBeansCPG committed:
add simple startup addon option

BroBeansCPG committed:
fix cargo unloading not unlocking seats

BroBeansCPG committed:
fix cargo unloading not unlocking seats

BroBeansCPG committed:
Merge remote-tracking branch 'origin/bugfix/miscJuly' into bugfix/miscJuly

Aaren882 committed:
add sound compat for AFM

Aaren882 committed:
Add APU startup sound (Int/Ext)
Fix MP Engine startup sound can't be heard if the bird isn't Local to 
the player
Tweak and Redo sounds (RotorBrake,EngineStart,EngineShutdown,cabin 
Door,CptDoor)

Aaren882 committed:
Fix Sound Cracks
Increase Obstruction and Occlusion factors

Aaren882 committed:
Balance engine sounds volume

Aaren882 committed:
Adjust Occlusion Obstruction Factors
Fix internal wind washing sound when getIn the bird
Tweak sound volume

Aaren882 committed:
Fix APU + Engine Sounds in MP (Tested on Dedicated Server)
 - Can be heard on other clients

Add wind-washing sound can be interacted by ACE Action
Optimize Net traffic and performance
Change Engine Sound Ext
Change APU distance Ext
Tweak Turbine Sound Ext

Aaren882 committed:
Tweak Sound Sources

Aaren882 committed:
Adjust Sound Balance
Make Sound more Directional

BroBeansCPG committed:
fix default laser code issue

BroBeansCPG committed:
Revert "fix default laser code issue"

This reverts commit 34daebd6a3a3b5025fda7710ac3623e37d679334.

Aaren882 committed:
Sounds crispier + directional + less overwhelming
Tweak Sound3Dprocessor
Sound Blance

Aaren882 committed:
Merge branch 'bugfix/miscJuly' of https://github.com/Project-Hatchet/H-60 into bugfix/miscJuly

Aaren882 committed:
Tweak Sound Setups

Aaren882 committed:
Change Ext Engine Noise

BroBeansCPG committed:
Revert "possible unstow fix"

This reverts commit 0bae633e4851ba27e1d937ff40dca7002229967b.

BroBeansCPG committed:
more robust laser code setup

BroBeansCPG committed:
fix helis hidden in zeus

BroBeansCPG committed:
qol build change

BroBeansCPG committed:
fix ESIS not working with simple startup

BroBeansCPG committed:
actually fix fuel discrep

BroBeansCPG committed:
fix another debug msg

BroBeansCPG committed:
temp fix for flir unstow/jvmf breaking

fix #420

Aaren882 committed:
Sound Tweaks
Optimize Sound Render
Add Internal Volume Multiplier (in case they're too loud for people)
Add Internal Rotor Sound on passenger seats
Fix External Environment Sound as Emitter

Aaren882 committed:
Fix JVMF compat from most used cTabs
TF_Ghost_1_4_1 :
　https://steamcommunity.com/sharedfiles/filedetails/?id=871504836
Devastator :
　https://steamcommunity.com/sharedfiles/filedetails/?id=2189592034
NSWDG :
　https://steamcommunity.com/sharedfiles/filedetails/?id=2511318948
1erGTD :
　https://steamcommunity.com/sharedfiles/filedetails/?id=2262006564

BroBeansCPG committed:
delete waypoint through fms

BroBeansCPG committed:
Merge remote-tracking branch 'origin/bugfix/miscJuly' into bugfix/miscJuly

BroBeansCPG commented:
Ready to merge

Aaren882 committed:
Balance APU Sound Volume
- Remove some unused sounds

Aaren882 committed:
Fix multiple play at the same time
- Remove **APU Gen** Updating APU state
- Use Say3D instead PlaySound3D to have better sounding
- some cleanUps

BroBeansCPG committed:
Merge branch 'Main' into bugfix/miscJuly

Aaren882 reviewed (commented):
These're for basic sound setups

Aaren882 committed:
Rework close rotor sound
Balance sounds at close range
Remove WAV files

Aaren882 committed:
Remove Sound folder (temporaly)

Aaren882 committed:
Add back Sound Files
Remove *.pkf (they're not needed)

Aaren882 committed:
Sound Adjustments :
- Sound Balance
- Apply Startup + Shutdown + APU Startup Sound s to new the sound engine
- Add APU Fuel Pump Delay
- heli sound can travel much further now
- Remove un-needed sound

BroBeansCPG committed:
fix monocle staying w/ hud off

derp

Aaren882 committed:
Minor optimization

BroBeansCPG reviewed (commented):
Reviewed
